### DISCUSSION TITLE
[SUGGESTION] GoBig may need these fixes

### DISCUSSION THREAD

**Discussion by WitheringCesspool** [hlky/stable-diffusion-webui#228] [Feature suggestions]
As you mentioned, GoBig is disabled currently due to a bug. I saw that someone on the stable diffusion discord (HankTheHillman) was able to get GoBig working by doing the following:

In the webui file, underneath the part where it said
                use_GoBIG=use_GoBIG,

They added:
                 gobig_strength=gobig_strength,
                gobig_steps=gobig_steps,

And there was a formatting mistake of a single underscore. 
They changed:

 samples_ddim, _ = K.sampling.__dict__[f'sample_{sampler.get_sampler_name()}'](model_wrap_cfg, xi, sigma_sched, extra_args={'cond': conditioning, 'uncond': unconditional_conditioning, 'cond_scale': cfg_scale}, disable=False)

to

            samples_ddim = K.sampling.__dict__[f'sample_{sampler.get_sampler_name()}'](model_wrap_cfg, xi, sigma_sched, extra_args={'cond': conditioning, 'uncond': unconditional_conditioning, 'cond_scale': cfg_scale}, disable=False)

(The indentations are probably a bit off). I'm not sure if this is the bug you're referring to but this solution was able to get GoBig working for them so thought I'd put it here in case it helps.

**Comment by dudumaroja** [comment]
it whould be nice we could use gobig in cpu mode if needed.. i cant use it in just 6gb ram sadly

**Reply by ack226** [reply]
Use the `--extra-models-cpu` option. That will force ESRGAN, GFPGAN, and GoBig to use CPU only. That's what I had to do to get them working on my 6GB card.

**Reply by dudumaroja** [reply]
i will try thanks
